frontend/src/types/item.d.ts

```diff
@@ -0,0 +1,15 @@
+export type Item = {
+    id: string,
+    name: string
+    slug: string
+    weight: Weight,
+    source?: string
+    image?: string
+    tags: Tag[]
```

Comment:
`user` & `createdAt` is missing

Check https://query.wwweights.com/swagger#/default/ItemsController_getItemList

I already reported defintions of return types in #269